src/prefect/orion/schemas/actions.py

```diff
@@ -31,159 +35,133 @@ def validate_block_document_name(value):
-            "description",
-            "work_queue_name",
-            "tags",
-            "manifest_path",
-            "path",
-            "entrypoint",
-            "parameters",
-            "storage_document_id",
-            "infrastructure_document_id",
-            "infra_overrides",
-        ],
-    )
-):
+@copy_model_fields
+class DeploymentUpdate(PrefectBaseModel):
     """Data used by the Orion API to update a deployment."""
 
+    version: Optional[str] = FieldFrom(schemas.core.Deployment)
+    description: str = FieldFrom(schemas.core.Deployment)
+    schedule: schemas.schedules.SCHEDULE_TYPES = FieldFrom(schemas.core.Deployment)
+    is_schedule_active: bool = FieldFrom(schemas.core.Deployment)
+    infra_overrides: Dict[str, Any] = FieldFrom(schemas.core.Deployment)
+    parameters: Dict[str, Any] = FieldFrom(schemas.core.Deployment)
+    tags: List[str] = FieldFrom(schemas.core.Deployment)
+    work_queue_name: Optional[str] = FieldFrom(schemas.core.Deployment)
+    path: str = FieldFrom(schemas.core.Deployment)
+    entrypoint: str = FieldFrom(schemas.core.Deployment)
+    manifest_path: str = FieldFrom(schemas.core.Deployment)
+    storage_document_id: Optional[UUID] = FieldFrom(schemas.core.Deployment)
+    infrastructure_document_id: Optional[UUID] = FieldFrom(schemas.core.Deployment)
+
     class Config:
         extra = "forbid"
 
 
-class FlowRunUpdate(
-    schemas.core.FlowRun.subclass(
-        name="FlowRunUpdate",
-        include_fields=[
-            "flow_version",
-            "parameters",
-            "name",
-            "tags",
-            "empirical_policy",
-        ],
-    )
-):
+@copy_model_fields
+class FlowRunUpdate(PrefectBaseModel):
     """Data used by the Orion API to update a flow run."""
 
+    name: str = FieldFrom(schemas.core.FlowRun)
+    flow_version: str = FieldFrom(schemas.core.FlowRun)
+    parameters: dict = FieldFrom(schemas.core.FlowRun)
+    empirical_policy: schemas.core.FlowRunPolicy = FieldFrom(schemas.core.FlowRun)
+    tags: List[str] = FieldFrom(schemas.core.FlowRun)
+
     class Config:
         extra = "forbid"
 
 
-class StateCreate(
-    schemas.states.State.subclass(
-        name="StateCreate",
-        include_fields=[
-            "type",
-            "name",
-            "message",
-            "data",
-            "state_details",
-        ],
-    )
-):
+@copy_model_fields
+class StateCreate(PrefectBaseModel):
     """Data used by the Orion API to create a new state."""
 
+    type: schemas.states.StateType = FieldFrom(schemas.states.State)
+    name: str = FieldFrom(schemas.states.State)
+    timestamp: schemas.core.DateTimeTZ = FieldFrom(schemas.states.State)
+    message: str = FieldFrom(schemas.states.State)
+    data: schemas.data.DataDocument = FieldFrom(schemas.states.State)
+    state_details: schemas.states.StateDetails = FieldFrom(schemas.states.State)
+
     class Config:
-        extra = "forbid"
+        extra = "ignore"
```

Comment:
This is the only real change here. This is necessary to support `StateCreate` in action models. Without this, I got errors that `StateCreate` does not accept ids. The client is just sending normal `State` objects. I believe that previously we actually were not actually overriding the `state` field type where we thought we were.